packages/dialtone-css/lib/build/less/components/link.less

```diff
@@ -127,4 +127,22 @@
             cursor: not-allowed;
         }
     }
+
+    //  $$  MENTION
+    //  ----------------------------------------------------------------------------
+    // Styling specific to mentions such as @brad.paugh. The underline highlighting
+    // is reversed compared to a regular link, and it has a light background.
+    &--mention {
+        color: var(--dt-color-link-primary);
+        line-height: var(--dt-font-line-height-200);
+        text-decoration: none;
+        background-color: hsl(var(--dt-color-purple-400-hsl) / 0.1);
```

Comment:
For some reason `--dt-color-link-primary` doesnt seem to be generating an hsl... using purple 400 for now.